templates/pages/account/orders/invoice.html

```diff
@@ -6,7 +6,7 @@
     <div class="Invoice">
         <h1>
             {{#if settings.store_logo.image}}
-                <img class="header-logo-image lazyload" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
+                <img class="header-logo-image lazyload" data-sizes="auto" src="{{getImage settings.store_logo.image}}" data-src="{{getImage settings.store_logo.image 'logo_size'}}" alt="{{settings.store_logo.title}}" title="{{settings.store_logo.title}}">
```

Comment:
So this is a lazy load image which will load the loading image first & then replace it will original image. Do we see anything in the console as to why the image is not loading ? Or else we completely remove lazy loading. See https://github.com/bigcommerce/cornerstone/blob/81d24b58633365ea1974e9d3e633111d66cc6d57/templates/components/common/store-logo.html

Reply:
I looked into that and find out that this issue happens because of the printing (the image is not ready by the time the printing windows opened.
I delete the lazy load and used the same method as referenced, however, the code in the reference use lazy load as well so I should modify it according to the requirements.